tests/e2e/server/routes.auth.mjs

```diff
@@ -226,12 +226,17 @@ export default function (app) {
 
   app.get(authPaths.authorize, wait, async (req, res) => {
     if (req.cookies.iPlanetDirectoryPro) {
-      const url = new URL(`${req.query.redirect_uri}`);
-      url.searchParams.set('client_id', 'bar');
-      url.searchParams.set('code', 'foo');
-      url.searchParams.set('iss', `${AM_URL}/oauth2`);
-      url.searchParams.set('state', req.query.state);
-      res.redirect(url);
+      // Is Central Login test
+      if (req.query.acr_values === 'SpecificTree') {
```

Comment:
This change is breaking the following e2e tests: `authn-oauth`, `config-token-storage` and `config-custom-paths`. They were failing with 400 status response and the error message about the missing arg_values.

Reply:
Ah, nice catch. Sorry for this. I've pushed an update.